docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -786,4 +786,136 @@ spec:
   tags: compliance, CIS, CIS1.4
   platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic security and data file updates is enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='CriticalUpdateInstall' AND value=1 LIMIT 1;
+  description: "Checks that the operating system is configured to automatically download updates to built-in macOS security tools such as malware removal tools."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic security and data update installation."
+  platforms: macOS
+  tags: compliance, CIS, CIS1.5
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic installation of operating system updates is enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticallyInstallMacOSUpdates' AND value=1 LIMIT 1;
+  description: "Checks that a mobile device management (MDM) solution configures the operating system to automatically install operating system updates."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic installation of operating system updates."
+  platforms: macOS
+  tags: compliance, CIS, CIS1.6
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Time and date are configured to be updated automatically (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.applicationaccess' AND name='forceAutomaticDateAndTime' AND value=1 LIMIT 1;
+  description: "Checks that a mobile device management (MDM) solution configures the operating system to automatically update the time and date."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic time and date configuration."
+  platforms: macOS
+  tags: compliance, CIS, CIS2.2.1
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Inactivity limit of 20 minutes or less (macOS)
+  query: SELECT 1 WHERE EXISTS (SELECT CAST(value as integer(4)) valueint from managed_policies WHERE domain = 'com.apple.screensaver' AND name = 'askForPasswordDelay' AND valueint <= 60 LIMIT 1) AND EXISTS (SELECT CAST(value as integer(4)) valueint from managed_policies WHERE domain = 'com.apple.screensaver' AND name = 'idleTime' AND valueint <= 1140 LIMIT 1) AND EXISTS (SELECT 1 from managed_policies WHERE domain='com.apple.screensaver' AND name='askForPassword' AND value=1 LIMIT 1);
+  description: "Checks that a mobile device management (MDM) solution configures the operating system lock the screen in 20 minutes or less."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables the screen saver and lock within an inactivity delay of 20 minutes or less."
```

Comment:
Profile vs. settings.

Reply:
We have so many of these policies that are looking at the MDM and not the underlying setting itself. How bad is that? Is it feasible to query against the underlying setting itself for some of these?

Reply:
> How bad is that? 

@zhumo I'm not sure how bad this is. 

However, it seems like this makes the argument that Fleet provides accurate data about the state of your devices (potential differentiator) weaker.

> Is it feasible to query against the underlying setting itself for some of these?

I think, for many of these, this isn't possible today. I think we need to add osquery tables to check the underlying setting for many of these.

cc @GuillaumeRoss

Reply:
I think it's fine for now, it's better to improve coverage first, and then, based on demand, we should see what osquery tables we could create. I'm not too worried about it, on Windows it's similar - CIS even says to check if a GPO is configuring X, instead of saying to check if X is configured!